hail/src/main/scala/is/hail/expr/ir/MatrixIR.scala

```diff
@@ -50,7 +50,7 @@ object MatrixIR {
     val newMatrixType = typ.copy(rvRowType = coerce[TStruct](rTyp.virtualType))
 
     val keepF = { (mv: MatrixValue, keep: Array[Int]) =>
-      val keepBc = mv.sparkContext.broadcast(keep)
+      val keepBc = mv.hc.sc.broadcast(keep)
```

Comment:
Was told that mv should have an instance of HailContext (or, at least for now, just the reference to the singleton). So all sparkContext calls become hc.sc

Reply:
Reverted to using sparkContext, for a lower-entropy change